examples/mp-openapi/README.adoc

```diff
@@ -0,0 +1,112 @@
+:index-group: MicroProfile
+:jbake-type: page
+:jbake-status: unpublished
+
+# Microprofile OpenAPI
+This is an example on how to use microprofile OpenAPI in TomEE.
+
+== Run the application:
+mvn clean install tomee:run 
+
+Within the application there is an endpoint that will give you weather based on day.
+
+== Request
+....
+GET http://localhost:8080/mp-openapi/weather/status/{today}
+
+
+== Response
+....
+{today} is a sunny day.
+
+
+== OpenAPI
+
+OpenAPI is a spec for natively producing OpenAPI v3 documents from JAX-RS applications. 
+There exists an endpoint /openApi which gives the OpenAPI document based on Accept request header.
+
+
+== Request
+....
+GET http://localhost:8080/mp-openapi/openapi

```

Comment:
It does not work without content-type as application/json so it is worth mention it. This is a bug in openapi I think. Because it should have a default message body reader/writer and it should be yaml. But it can be fixed later, there is a discussion on the list about it.

Reply:
I think it is working as expected now, without the Accept:application/json header it is outputting a valid YAML. I think I should document that in the README file.

Reply:
I still get "No message body writer has been found for class org.apache.geronimo.microprofile.openapi.impl.model.OpenAPIImpl, ContentType: text/html" without the accept header.

Reply:
It is a bug, so you just need to add the header to the documentation. And if possible add a ticket in the Geronimo JIRA.

Reply:
I think the browser by default sends ContentType: text/html as header if I do the same request in Postman I get a YAML, so I dont think a ticket is required.

Reply:
Ok I opened a PR in geronimo to tackle this https://github.com/apache/geronimo-openapi/pull/4.